rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -968,6 +981,8 @@ def load(
             )
             return cls(component_config=meta)
 
+        finetune_mode = kwargs.pop("should_finetune", False)
```

Comment:
I'd add this to the method parameters so you can directly access it instead of using `kwargs` and the string.